.github/workflows/docs.yml

```
@@ -25,7 +30,8 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        persist-credentials: false
```

actionlint (1.7.12)

[error] unexpected key "persist-credentials" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)